cityofaustin/atd-vz-data
# Add new near trauma center fields to crashes table

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15279

I couldn't find anything about the new fields in the [CRIS release notes](https://austininnovation.slack.com/archives/CM9MK950S/p1705084971737399?thread_ts=1705084752.767399&cid=CM9MK950S), I wonder where else we would find info on added fields? I figured out what fields we needed to add by looking at the`record_json` in the `atd_txdot_changes` table and it was just these two having to do with nearby trauma centers.

Follow up question to this PR is -- are these fields that would be of interest to the VZ team and should we display them in the VZE on the Crash details page?

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch locally, replicate a fresh copy of prod data & start up the graphql engine.
2. Start VZE locally and observe how if you go to the Changes tab, click on a record, select some fields, and try to "Save Selected Changes" like it says in the issue, you will get a graphql error.
3. Now run the up migrations.
```
hasura migrate apply --up 1
hasura metadata apply
```
5. Try to "Save Selected Changes" on some records in the Changes tab now, you should get no errors!
6. Test running the down migration
`hasura migrate apply --down 1`
7. Test editor user, you should not see the Changes tab in the nav bar anymore.


---
#### Ship list
